pkg/registry/controller/rest.go

```diff
@@ -47,14 +48,26 @@ func (rcStrategy) NamespaceScoped() bool {
 func (rcStrategy) PrepareForCreate(obj runtime.Object) {
 	controller := obj.(*api.ReplicationController)
 	controller.Status = api.ReplicationControllerStatus{}
+	controller.Spec.SequenceNumber = 0
+	controller.Spec.SequenceNumber = 0
```

Comment:
oh-- gotcha. I think it's not important to do that though-- does it cause any harm if it starts out with seq 42?